functional_tests.go

```diff
@@ -5641,9 +5643,6 @@ func testPresignedPostPolicy() {
 		"policy": "",
 	}
 
-	// Seed random based on current time.
-	rand.Seed(time.Now().Unix())
```

Comment:
Deprecated, we can remove this call from every test, I might make a follow up PR to do the rest.